tasks/zadanie_20.php

```diff
@@ -1,27 +1,12 @@
 <?php
 
-/*
-Napisz program, który obliczy cenę brutto na podstawie stawki VAT zależnej od kategorii produktu. Użyj instrukcji match.
-Stawki VAT (zmyślone) dla kategorii:
-- elektronika: 22%
-- odzież: 22%
-- żywność: 8%
-- książki: 5%
-- czasopisma: 5%
-- pozostałe: 23%
-
-Zmienne:
-$net - cena netto
-$category - kategoria produktu
-
-Przykład:
-
-$net = 100;
-$category = 'odzież';
-
-Cena brutto wynosi 122 zł.
-
-*/
-
 $net = $params[0]; // tej linijki nie ruszamy :)
 $category = $params[1]; // tej linijki nie ruszamy :)
+
+$summary=match($category){
+    "elektronika" , "odzież" => $net * 1.22,
+    "książki" , "czasopisma" => $net * 1.05,
+    "żywność" => $net * 1.08,
+    default => $net * 1.23,
+};
+echo "Cena brutto wynosi $summary zł."
```

Comment:
> PHP Parse error:  syntax error, **unexpected end of file, expecting "," or ";"** in /home/runner/work/shoper-up/shoper-up/tasks/zadanie_20.php on line 12

Tutaj brakuje średnika na końcu ;)
```suggestion
echo "Cena brutto wynosi $summary zł.";
```